clambcc/clambc-compiler.py

```diff
@@ -574,23 +784,56 @@ def optimize(clangLLVM: ClangLLVM, inFile: str, outFile: str, sigFile: str, inpu
                                              #otherwise the writer gets
                                              #unhappy.
           f' -globalopt'
-          f' -clambc-convert-intrinsics'   #convert all memset intrinsics to
+          f' -clambc-convert-memsets-to-32Bit'   #convert all memset intrinsics to
                                           #the 32-bit instead of the 64-bit
                                           #intrinsic
           f' -clambc-writer'               #write the bytecode
           f' -clambc-writer-input-source={inputSourceFile}'
           f' -clambc-sigfile={sigFile}'
           )
+    """
+
+    """
+    cmd = f"{clangLLVM.getOpt()} %s -o %s %s %s --passes=\"%s\" -clambc-writer-input-source=%s -clambc-sigfile=%s" % (
+            inFile
+             , outFile
+             , " ".join(OPTIMIZE_OPTIONS)
+             , " ".join(OPTIMIZE_LOADS)
+             , ",".join(OPTIMIZE_PASSES)
+             , inputSourceFile
+             , sigFile
+             )
+    """
+
+    cmd = []
+    cmd.append(clangLLVM.getOpt())
+    cmd.append(inFile)
+    cmd.append('-o')
+    cmd.append(outFile)
+    cmd += OPTIMIZE_OPTIONS
+    cmd += OPTIMIZE_LOADS
+
+    s = '--passes="'
+    first = True
+    for v in OPTIMIZE_PASSES:
+        if first:
+            first = False
+        else:
+            s += ','
+        s += v
+    s += '"'
+    cmd.append(s)
+
 
-    if standardCompiler:
-        cmd += f" -clambc-standard-compiler"
+    cmd.append(f'-clambc-writer-input-source={inputSourceFile}')
+    cmd.append(f'-clambc-sigfile={sigFile}')
 
-    return run(cmd)
+    return runNew(cmd)
 
 
-def genExe(clangLLVM: ClangLLVM, optimizedFile: str, outputFile: str) -> int:
-    cmd = f"{clangLLVM.getClang} {optimizedFile} -o {outputFile}"
-    return run(cmd)
+#def genExe(clangLLVM: ClangLLVM, optimizedFile: str, outputFile: str) -> int:
+#    cmd = f"{clangLLVM.getClang} {optimizedFile} -o {outputFile}"
+#    return run(cmd)
```

Comment:
More dead cod here and down below.

Reply:
Removed that and the options that used it.  They were some debugging ideas I had that weren't helping, so I never finished them.